### PR TITLE
Adding proposal clarifications

### PR DESCRIPTION
**Pull request by VoteWizard**

### CODE REVIEW

File: governance/votes/Executive vote - September 13, 2024.md [hunk 1]
```diff
@@ -35,12 +35,28 @@ If this executive proposal passes, the following **actions** will occur within t
...
+## Proposal Clarifications
+
+### Execution date restriction
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Poll 1134](https://vote.makerdao.com/polling/QmTySKwi)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/1)
+
+This executive proposal has its earliest execution date set to **September 17, 2024 at 12:00 UTC**. This means that it cannot be executed before that date.
+
+### The ESM will not be authorized in any new contracts deployed for this proposal
```
Comment (VoteWizard) [inline governance/votes/Executive vote - September 13, 2024.md:51]:
```suggestion
### New Contract ESM Exclusion
```

File: governance/votes/Executive vote - September 13, 2024.md [hunk 2]
```diff
@@ -35,12 +35,28 @@ If this executive proposal passes, the following **actions** will occur within t
...
+
+- **Authorization**: [Ecosystem Approval](TBD)
+- **Proposal**: [Forum Post](http://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/3)
+
+The ESM will **not** be authorized in any new contracts deployed for this proposal.
```
Comment (VoteWizard) [inline governance/votes/Executive vote - September 13, 2024.md:56]:
```suggestion
The [Emergency Shutdown Module (ESM)](https://docs.makerdao.com/smart-contract-modules/shutdown/emergency-shutdown-module) will **not** be authorized in any new contracts deployed for this proposal.
```

**Review by MakerJanSky — COMMENTED** [review]
Looking good, VoteWizard. I will approve this PR once Ecosystem Approval for the New Contract ESM Exclusion has been linked.

File: governance/votes/Executive vote - September 13, 2024.md [hunk 3]
```diff
@@ -172,7 +188,7 @@ If this executive proposal passes, then DssVestMintable for SKY will be set up t
 
 ### USDS->SKY Farming Setup
 
-- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Poll 1134](https://vote.makerdao.com/polling/QmTySKwi)
+- **Authorization**: [Ecosystem Approval 1](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Ecosystem Approval 2](TBD), [Poll 1134](https://vote.makerdao.com/polling/QmTySKwi) ,[Poll 1134](https://vote.makerdao.com/polling/QmTySKwi)
```
Comment (VoteWizard) [inline governance/votes/Executive vote - September 13, 2024.md:191]:
```suggestion
- **Authorization**: [Ecosystem Approval 1](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Ecosystem Approval 2](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/4), [Poll 1134](https://vote.makerdao.com/polling/QmTySKwi) ,[Poll 1134](https://vote.makerdao.com/polling/QmTySKwi)
```

File: governance/votes/Executive vote - September 13, 2024.md [hunk 4]
```diff
@@ -35,12 +35,28 @@ If this executive proposal passes, the following **actions** will occur within t
...
+### Execution date restriction
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Poll 1134](https://vote.makerdao.com/polling/QmTySKwi)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/1)
+
+This executive proposal has its earliest execution date set to **September 17, 2024 at 12:00 UTC**. This means that it cannot be executed before that date.
+
+### New Contract ESM Exclusion
+
+- **Authorization**: [Ecosystem Approval](TBD)
```
Comment (VoteWizard) [inline governance/votes/Executive vote - September 13, 2024.md:53]:
```suggestion
- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/4)
```

File: governance/votes/Executive vote - September 13, 2024.md [hunk 5]
```diff
@@ -172,7 +188,7 @@ If this executive proposal passes, then DssVestMintable for SKY will be set up t
 
 ### USDS->SKY Farming Setup
 
-- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Poll 1134](https://vote.makerdao.com/polling/QmTySKwi)
+- **Authorization**: [Ecosystem Approval 1](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Ecosystem Approval 2](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/4), [Poll 1134](https://vote.makerdao.com/polling/QmTySKwi) ,[Poll 1134](https://vote.makerdao.com/polling/QmTySKwi)
```
Comment (VoteWizard) [inline governance/votes/Executive vote - September 13, 2024.md:191]:
```suggestion
- **Authorization**: [Ecosystem Approval 1](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/2), [Ecosystem Approval 2](https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031/4), [Poll 1134](https://vote.makerdao.com/polling/QmTySKwi)
```

**Review by MakerJanSky — APPROVED** [review]
Nice work, all.

**Pull request merged**